src/filter.js

```diff
@@ -252,29 +254,60 @@ export class Filter extends CriteriaBuilder {
   }
 
   generateFields(columns, entityName) {
-    let excludeColumns = (this.excludeColumns) ? this.excludeColumns.replace(/\s/g, '').split(',') : [];
+    let paths = [];
 
-    if (this.showIdColumns) {
-      columns.id = 'number';
+    let excludeColumns = splitOnColumnOrIdentity(this.excludeColumns);
+    let includeColumns = splitOnColumnOrIdentity(this.includeColumns);
+
+    if (excludeColumns && includeColumns) {
+      excludeColumns = [];
+
+      logger.warn(`both excludeColumns and includeColumns bindables are defined, ignoring excludeColumns`);
```

Comment:
When it comes to sensitive data it's much harder to explicitly expose that data then to implicitly expose it.

If you put the password property in the include bindable you propably want to show the password. If you forget to put password in exclude you might have made a mistake.

So explicit over implicit. This include should get priority.